massgov/mayflower
# DP-17150 Relationship Indicator

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

- Separate the relationship indicator changes from the global nav feature branch `patternlab/DP-17138_globalnav-relationship-indicator` as a new branch from the latest `develop`.
- Change the label for the secondary set to **OFFERED BY**.
- Change the widths to:
         - Primary - 60%
         - Secondary - 40%
- Remove the 15 px padding at the bottom from the primary set container to match the secondary set's style.
- Fix the keyboard trap on the show more/less button WHEN THE HIDDEN ITEMS ARE DISPLAYED.


## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-17150)

## Steps to Test


1. Check the following changes in http://localhost:3000/?p=pages-service.

- "Offered by" and Organizations should be on the right in green.  Switch the actual text shown so that "Department of Conservation and Recreation" and the "Other agency..." text are on the right with the header "Offered by".
- "Related to" and the 6 items shown should be on the left and in blue.  Change "Related to" to "Part of"
- Tabbing is trapped when "show more" in the relationship indicator is expanded.  User should be able to tab out of it.
- There should be less space between label ("Offered by" and "Part of") and data. See design linked below.
- Left side containing "Part of" should take up roughly 60% of the total screen width as shown rather than the current width which appears 50%.
- This div `<div class="ma__relationship-indicators--section primary">` has 15px of padding in desktop.  It should not - this is needed only when the relationship indicators are stacked up on a mobile screen.  The padding should match the secondary on a desktop and should be 1 px.
- No keyboard trap around the show more/less button .

## Screenshots
<img width="434" alt="Pattern_Lab_-_pages-service" src="https://user-images.githubusercontent.com/9633303/73276121-2e02a500-41b6-11ea-8627-1af3ba99ca45.png">

<img width="786" alt="Pattern_Lab_-_pages-service_1" src="https://user-images.githubusercontent.com/9633303/73276122-2e02a500-41b6-11ea-9f65-f7d88478eb97.png">



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
